lodash.js

```diff
@@ -3724,7 +3724,7 @@
 
       var result = baseMap(collection, function(value, key, collection) {
         var criteria = arrayMap(iteratees, function(iteratee) {
-          return iteratee(value);
+          return { 'computed': false, 'iteratee': iteratee, 'element': value };
```

Comment:
You can use this syntax:
```js
{ 'computed': false, iteratee, 'element': value }

Reply:
It's ok, we'll switch to `terser` instead if `uglify`.

Reply:
I’m not sure what you’re asking—would you like me to switch this to use
the punning syntax, such that `npm test` fails at this commit, or should
I leave it as is? (Or wait for an update to the base branch 4.17.12-pre
to change the parser?)